### ISSUE TITLE
[Comprehension] User can't perceive an element.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#80595]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Benefit Tools
**Product:** Claim Status Tool
**Feature:** Improved Evidence Request Initiative

## Findings details
**VA.gov Experience Standard - issue:** User can't perceive an element.
**VA.gov Experience Standard - category:** Comprehension
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In the "What you need to do" section on claims, if an item has a deadline in the future the "Needed from you by..." text is in black text and if the deadline is in the past the "Needed from you by..." text is in dark red. (With test user Kyle Cole/+23, compare [/your-claims/600494815/status](https://staging.va.gov/track-claims/your-claims/600494815/status) and [/your-claims/600471552/status](https://staging.va.gov/track-claims/your-claims/600471552/status)).

It looks like the change in text color is the only thing indicating the deadline status --- upcoming versus past due. This distinction is likely to be missed by many colorblind users.

<details><summary>Link, screenshot or steps to recreate</summary>

It looks like these just get a `past-due` class when they reach that status:
```
<div class="due-date-header">
	<strong class="past-due">Needed from you by April 7, 2024</strong>
</div>
```

</details>

## Recommended action

Recommend inserting text whenever you hit the past due status. On the /document-request/ page for that action, an icon is inserted along with the text "due 4 days ago" so something similar seems appropriate here. 

## References

- **WCAG Success Criteria:** [WCAG 1.4.1 Use of Colour - Level A](https://www.w3.org/TR/WCAG21/#use-of-color)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by skylerschain** [comment]
@briandeconinck Do you happen to have a screenshot of this? Or do you know which testing user you were using? I know what state you're referring to, but I'm not sure which testing account would have this off the top of my head.

cc: @jerekshoe or @pmclaren19 in case you know

**Comment by skylerschain** [comment]
Mocks for this [here](https://www.figma.com/file/F8U4wddaFouUPVd4mGBMDI/CST---Evidence-Submission---Spec?type=design&node-id=1719-10358&mode=design&t=5BAgj0mxRi1iUTIe-4).

**Comment by pmclaren19** [comment]
@skylerschain should we format it so that it looks the same as the document request? 
![Screenshot 2024-04-16 at 3 08 42 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/141954992/e6c5756f-fdf5-4f82-bc5f-dbb93bac61c7)

Instead of 'Needed from you by {DATE} - Due {DUE_DATE_FROM_NOW}' we'd do 'Needed from you by {DATE} - due {DUE_DATE_FROM_NOW}'

**Comment by skylerschain** [comment]
Let's capitalize the "D" in "Due" for both of them.

**Comment by pmclaren19** [comment]
Met with @skylerschain this morning and we are going to update the Document Request page so that it uses the same due date header as the Primary Alert (aka FilesNeeded component). 

When a due date is past due the header will read: `Needed from you by {DATE} - Due {# of days, months, years} ago `and will show in red text.

When a due date has not been met yet the header will read: `Needed from you by {DATE}` and will show in black text.

**Comment by pmclaren19** [comment]
Tested/ verified in Staging and looks good!
